buildscripts/azure/template-linux-macos.yml

```diff
@@ -30,12 +30,9 @@ jobs:
       continueOnError: ${{ parameters.allowFailure }}
 
     - script: |
-        if [ "$USE_NUMBA_MASTER" == "False" ]; then
-            $HOME/miniconda3/bin/python buildscripts/test.py --test-mode=conda --python=$PYTHON_VER --numpy=$NUMPY_VER --conda-prefix=$HOME/miniconda3;
-        else
-            $HOME/miniconda3/bin/python buildscripts/test.py --test-mode=conda --python=$PYTHON_VER --numpy=$NUMPY_VER --use-numba-master --conda-prefix=$HOME/miniconda3;
-        fi
-      displayName: 'Test conda package'
+        source $HOME/miniconda3/bin/activate
+        python buildscripts/test.py --python=$PYTHON_VER --sdc-channel=./sdc-build
```

Comment:
Are you sure what all commands are executed in a single shell? If not - need to activate conda for each command.

Reply:
Yes, it is ok